core/src/main/java/io/questdb/griffin/SqlCodeGenerator.java

```diff
@@ -2936,6 +2951,12 @@ private RecordCursorFactory generateTableQuery(
         }
     }
 
+    private boolean isOrderDescendingByDesignatedTimestampOnly(QueryModel model) {
+        return model.getOrderByAdvice().size() == 1 && model.getTimestamp() != null &&
+                Chars.equals(model.getOrderByAdvice().getQuick(0).token, model.getTimestamp().token) &&
```

Comment:
how does this work with case-insensitive SQL ?

```sql
select ts from tab order by TS
```

Reply:
In the following cases it looks like both values are lowercase at this point  : 

select * from trips order by pickup_datetimE DESC limit 100;

select * from trips order by pickup_DATETIME DESC limit 100; 

select * from trips order by PICKUP_datetimE DESC limit 100;   

select * from trips order by PICKUP_DATETIME DESC limit 100;

select cab_type, vendor_id, pickup_datetime as pick_up_datetime from trips order by PICKUP_datetimE DESC limit 100;

select cab_type, vendor_id, pickup_datetime  pick_up_datetime from trips order by PICKUP_datetimE DESC limit 100;

select cab_type, vendor_id, pick_up_datetime
from 
( select  cab_type, vendor_id, pickup_datetime as pick_up_datetime, rate_code_id from trips order by PICKUP_datetimE DESC limit 100 )

I adjusted SelectedRecordCursorFactory.hasDescendingOrder() to skip DESC ordering for the last 3 queries (it was disabled by default implementation ) .

Reply:
can we have unit tests proving case-insensitivity of this particular execution flow?

Reply:
I added tests  to OrderByDescRowSkippingTest .